packages/plugin-vsl-animate-occlusion/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "@jspsych-contrib/plugin-vsl-animate-occlusion",
+  "version": "1.0.0",
```

I forgot that we should use 0.1.0 here so that changesets can do the initial 1.0.0 release (once I set that up here)...
```suggestion
  "version": "0.1.0",
```